test/fill.js

```diff
@@ -1,4 +1,4 @@
-var cwise = require("cwise")
+var cwise = require("..")
```

Comment:
I'm interesting in knowing why this line needed to be patched.

Reply:
Wouldn't this have been testing the npm-distributed copy rather than the local code? Seems a bit fishy.

Reply:
Ohh, unless it's required to get static-module to correctly locate and replace `require('cwise')`…

Reply:
Ohhh, I get it. It aliases `node_modules/cwise` to `..`, probably so that static module works correctly. That means `npm run pretest` is required before `npm run test`.

Reply:
Relevant line here: https://github.com/scijs/cwise/blob/master/package.json#L23

Reply:
Thanks for investigating @rreusser :microscope: 

> It aliases node_modules/cwise to .., probably so that static module works correctly.

It would be nice to have someone confirming this.

Reply:
That line with pretest is os dependent but is obsolete with '..'

Reply:
@hakandilek I think you might be right.

Reasons I think it's an issue: this extra indirection is present, which suggests it may be needed. [static-module](https://github.com/browserify/static-module) is [a dependency](https://github.com/scijs/cwise/blob/master/package.json#L13) and treats `require('..')` differently from `require('cwise')`, even if they resolve to the same thing.

Reasons I question whether it's an issue: it doesn't immediately jump out to me that the tests are actually using static-module.

Seems alright to me to use `require('..')` if the tests pass and if linking this to and testing this with a real-world project succeeds.